boss-ws/src/main/java/org/genomebridge/boss/http/objectstore/FCSObjectStore.java

```diff
@@ -51,5 +51,10 @@ public boolean exists(String objKey) {
         return response.getStatus() == Response.Status.OK.getStatusCode();
     }
 
+    @Override
+    public URI generateResumableUploadURL(String objectName) {
+        throw new ObjectStoreException("Resumable upload is not currently supported on FCS storage.");
```

Comment:
Need to implement this method to make unit test pass (once other things get fixed).  Implementation is just:
 return generateResolveURI(objKey,HttpMethod.PUT,0L,null,null);